ThunkLibs/Generator/gen.cpp

```diff
@@ -761,12 +761,29 @@ void GenerateThunkLibsAction::EmitOutput() {
           version_suffix = '.' + std::to_string(*lib_version);
         }
         const std::string library_filename = libfilename + ".so" + version_suffix;
-        file << "  fexldr_ptr_" << libname << "_so = dlopen(\"" << library_filename << "\", RTLD_LOCAL | RTLD_LAZY);\n";
+
+        // Load the host library in the global symbol namespace.
+        // This follows how these libraries get loaded in a non-emulated environment,
+        // Either by directly linking to the library or a loader (In OpenGL or Vulkan) putting everything in the global namespace.
+        file << "  fexldr_ptr_" << libname << "_so = dlopen(\"" << library_filename << "\", RTLD_GLOBAL | RTLD_LAZY);\n";
 
         file << "  if (!fexldr_ptr_" << libname << "_so) { return false; }\n\n";
         for (auto& import : thunked_api) {
+          if (import.host_loader == "dlsym") {
```

Comment:
Could this be done by adding a utility function to `Host.h` instead and changing the `host_loader` default to that function?

This avoids specializing code generator logic for specific symbols and hence reduces overall generator complexity.

Reply:
Changed over to a utility function.